### PR TITLE
When logging redirect back to the original page that the user was trying...

### PR DESCRIPTION
**Pull request by jkimbo**
... to view

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Flask==0.10.1
 Flask-Admin==1.0.8
 Flask-SQLAlchemy==1.0
-GitHub-Flask==0.3.4
+GitHub-Flask==2.0.1
```
Comment (davidszotten) [inline requirements.txt]: this is a big jump. have you reviewed the changelog?

**Comment by davidszotten** [comment]
i guess this currently has no test coverage? fancy adding something basic?

File: requirements.txt (hunk 1 above)
Comment (jkimbo) [inline requirements.txt]: There isn't a changelog unfortunately but it doesn't look like that much has changed: https://github.com/cenkalti/github-flask/compare/7197116b27fe6b2ba694e49d6d2b54698b5e1505...master

Actually it's dropped the requirement for the `GITHUB_CALLBACK_URL` variable so I'll remove it from the env

**Comment by jkimbo** [comment]
Added some tests though they are fairly menial

**Comment by davidszotten** [comment]
:+1:

**Pull request merged**